pkg/asset/tls/helper.go

```diff
@@ -0,0 +1,109 @@
+package tls
+
+import (
+	"crypto/x509/pkix"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/types"
+)
+
+const (
+	validityTenYears      = time.Hour * 24 * 365 * 10
+	validityThirtyMinutes = time.Minute * 30
```

Comment:
nit: I'd make these public, and then import them int `installer/pkg/config-generator` (where we currently [define them locally][1]).  But I'm fine punting on that to follow-up work to help get this PR landed more quickly.

[1]: https://github.com/openshift/installer/blob/aaa47f6a54f0fa0449d36de01ccb9f56729b608a/installer/pkg/config-generator/tls.go#L52-L53